### ISSUE TITLE
Load Error for nokogiri/nokogiri

### ISSUE THREAD

**Issue by Weltraumschaf** [barthel/docker-asciidoctor#1]
When I simply invoke the container:

```shell
docker run --rm -v $(pwd):/documents registry.iteragit.iteratec.de/ubarthel/documentation:latest
```

I got this stack trace:

```
<internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:159:in `require': cannot load such file -- nokogiri/nokogiri (LoadError)
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:159:in `rescue in require'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:39:in `require'
        from /usr/lib/ruby/gems/3.2.0/gems/nokogiri-1.15.4-aarch64-linux/lib/nokogiri/extension.rb:31:in `rescue in <top (required)>'
        from /usr/lib/ruby/gems/3.2.0/gems/nokogiri-1.15.4-aarch64-linux/lib/nokogiri/extension.rb:4:in `<top (required)>'
        from /usr/lib/ruby/gems/3.2.0/gems/nokogiri-1.15.4-aarch64-linux/lib/nokogiri.rb:8:in `require_relative'
        from /usr/lib/ruby/gems/3.2.0/gems/nokogiri-1.15.4-aarch64-linux/lib/nokogiri.rb:8:in `<top (required)>'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from /usr/lib/ruby/gems/3.2.0/gems/gepub-1.0.15/lib/gepub/meta.rb:2:in `<top (required)>'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from /usr/lib/ruby/gems/3.2.0/gems/gepub-1.0.15/lib/gepub.rb:8:in `<top (required)>'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from /usr/lib/ruby/gems/3.2.0/gems/asciidoctor-epub3-1.5.1/lib/asciidoctor-epub3.rb:5:in `<top (required)>'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from /usr/lib/ruby/gems/3.2.0/gems/asciidoctor-epub3-1.5.1/bin/asciidoctor-epub3:5:in `<top (required)>'
        from /usr/bin/asciidoctor-epub3:25:in `load'
        from /usr/bin/asciidoctor-epub3:25:in `<main>'
<internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require': cannot load such file -- nokogiri/nokogiri (LoadError)
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from /usr/lib/ruby/gems/3.2.0/gems/nokogiri-1.15.4-aarch64-linux/lib/nokogiri/extension.rb:31:in `rescue in <top (required)>'
        from /usr/lib/ruby/gems/3.2.0/gems/nokogiri-1.15.4-aarch64-linux/lib/nokogiri/extension.rb:4:in `<top (required)>'
        from /usr/lib/ruby/gems/3.2.0/gems/nokogiri-1.15.4-aarch64-linux/lib/nokogiri.rb:8:in `require_relative'
        from /usr/lib/ruby/gems/3.2.0/gems/nokogiri-1.15.4-aarch64-linux/lib/nokogiri.rb:8:in `<top (required)>'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from /usr/lib/ruby/gems/3.2.0/gems/gepub-1.0.15/lib/gepub/meta.rb:2:in `<top (required)>'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from /usr/lib/ruby/gems/3.2.0/gems/gepub-1.0.15/lib/gepub.rb:8:in `<top (required)>'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from /usr/lib/ruby/gems/3.2.0/gems/asciidoctor-epub3-1.5.1/lib/asciidoctor-epub3.rb:5:in `<top (required)>'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from /usr/lib/ruby/gems/3.2.0/gems/asciidoctor-epub3-1.5.1/bin/asciidoctor-epub3:5:in `<top (required)>'
        from /usr/bin/asciidoctor-epub3:25:in `load'
        from /usr/bin/asciidoctor-epub3:25:in `<main>'
/usr/lib/ruby/gems/3.2.0/gems/nokogiri-1.15.4-aarch64-linux/lib/nokogiri/extension.rb:7:in `require_relative': Error loading shared library ld-linux-aarch64.so.1: No such file or directory (needed by /usr/lib/ruby/gems/3.2.0/gems/nokogiri-1.15.4-aarch64-linux/lib/nokogiri/3.2/nokogiri.so) - /usr/lib/ruby/gems/3.2.0/gems/nokogiri-1.15.4-aarch64-linux/lib/nokogiri/3.2/nokogiri.so (LoadError)
        from /usr/lib/ruby/gems/3.2.0/gems/nokogiri-1.15.4-aarch64-linux/lib/nokogiri/extension.rb:7:in `<top (required)>'
        from /usr/lib/ruby/gems/3.2.0/gems/nokogiri-1.15.4-aarch64-linux/lib/nokogiri.rb:8:in `require_relative'
        from /usr/lib/ruby/gems/3.2.0/gems/nokogiri-1.15.4-aarch64-linux/lib/nokogiri.rb:8:in `<top (required)>'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from /usr/lib/ruby/gems/3.2.0/gems/gepub-1.0.15/lib/gepub/meta.rb:2:in `<top (required)>'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from /usr/lib/ruby/gems/3.2.0/gems/gepub-1.0.15/lib/gepub.rb:8:in `<top (required)>'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from /usr/lib/ruby/gems/3.2.0/gems/asciidoctor-epub3-1.5.1/lib/asciidoctor-epub3.rb:5:in `<top (required)>'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from /usr/lib/ruby/gems/3.2.0/gems/asciidoctor-epub3-1.5.1/bin/asciidoctor-epub3:5:in `<top (required)>'
        from /usr/bin/asciidoctor-epub3:25:in `load'
        from /usr/bin/asciidoctor-epub3:25:in `<main>'
```

Environment:
- macOS 14.3 (23D56)
- CPU Apple M1
- Rosetta 2 installed
- Docker version 25.0.1, build 29cf629222
- colima version 0.6.7 (no Docker Desktop)

**Comment by barthel** [comment]
https://github.com/sparklemotion/nokogiri/issues/2430
https://nokogiri.org/tutorials/installing_nokogiri.html#other-installation-scenarios

**Comment by Weltraumschaf** [comment]
Should I do something?

**Comment by barthel** [comment]
@Weltraumschaf 
No need for action. I have only collected information about this error.

Fix is still in progress.

**Comment by barthel** [comment]
Should be fixed with https://github.com/asciidoctor/docker-asciidoctor/pull/407 and resolved in release [1.63.0](https://github.com/asciidoctor/docker-asciidoctor/releases/tag/1.63.0) of the original Asciidoctor image.

**Comment by barthel** [comment]
This should be fixed with https://github.com/barthel/docker-asciidoctor/releases/tag/1.63.0, as we are using the original image from Asciidoctor as the base image from this release onwards.
